bin/parse_phonon.py

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/env -S python -B
+
+# Copyright IBM Corp. 2024
+# SPDX-License-Identifier: Apache2.0
+
+import argparse
+import os
+import sys
+import numpy as np
+
+from modules.calculate_properties import (get_CellParameters,
+                                          get_AtomicPositions,
+                                          calculate_UnitCells,
+                                          get_spg_class,
+                                          get_forces,
+                                          get_pol_tensor,
+                                          lorentzian)
+
+from modules.io_files import (save_axsf)
+
+from modules.constants import factor2cm
+
+from phonopy import Phonopy
+from phonopy.structure.atoms import PhonopyAtoms
+
+from phonopy.units import CP2KToTHz
+
+from ase.cell import Cell
```

Comment:
If one were to call the "Organize imports" function of VS Code on these files, could we get a systematic ordering and grouping of these imports?

Reply:
Done